### DISCUSSION TITLE
Seeing a difference in color rendering between nightly and stable, is this intentional?

### DISCUSSION THREAD

**Discussion by yoav-lavi** [wezterm/wezterm#4803] [Q&A]
### What Operating System(s) are you running on?

macOS Sonoma 14.2.1 (M1 Pro)

### WezTerm version

`wezterm 20230712-072601-f4abf8fd`

### Ask your question!
Seeing a difference in color rendering between the stable version and nightly, wanted to check if this is intentional:

Stable:
<img width="506" alt="Screenshot 2024-01-17 at 14 38 34" src="https://github.com/wez/wezterm/assets/14347895/fb5b115c-9ad1-42e8-ac45-a5b44e42f2ac">
Nightly:
<img width="676" alt="Screenshot 2024-01-17 at 14 38 27" src="https://github.com/wez/wezterm/assets/14347895/6cb0b492-947a-41a8-ae0a-46b32dbe8ac3">

Note the cyan color for `darwin` (in the prompt), it's lighter on Nightly. Seems a bit less clear in the image but should still be visible.

Thanks!

**Comment by wez** [comment]
likely because nightly defaults `front_end="WebGpu"`.  Try explicitly setting it in your config.  https://wezfurlong.org/wezterm/config/lua/config/front_end.html
